SCOPE.md

```diff
@@ -36,6 +36,12 @@ The badges API distracts from the main focus and complexity of the proposed spec
 
 It's likely that badges, or some way of retrieving lists of entities would be included as a separate endpoint.
 
+### Response links
+
+Many REST APIs link to endpoints around them.
+The Mediawiki core team suggested that they may also want to do this.
+It is likley that we will investigate this in the future, but for this this falls out of scope for the initial spec.
```

Comment:
Good to mention, thank you. The paragraph did feel a bit disjointed though, how about something like this:

```suggestion
While many REST APIs link to endpoints around them, unfortunately it currently falls out of scope for this initial API specification. We do intend to investigate this topic for future iteration of this spec, in accordance with recommendations from the Mediawiki Core Platform Team.
```